**Az első 5 feladat ismétlés (többtáblás lekérdezések) a tanulmányi adatbázisból**

1. Készítsünk listát arról, hogy melyik óra melyik teremben van!
    
    1. Jelenítsük meg az óra azonosítóját és a terem nevét!
    2. A listát szűrjük a 3 karakter hosszú, \[100;199\] zárt intervallumba eső termekre! Feltesszük, hogy a 3 karakter hosszú terem nevek csak számjegyeket tartalmaznak.

In [3]:
SELECT ora.ora_id AS 'Óra azonosító',
       ter.Terem AS 'Terem'
FROM Orak ora JOIN Termek ter ON ora.terem = ter.Tkód

(67 rows affected)

Total execution time: 00:00:00.007

ora_id tanar nap Sav targy terem Tkód Terem 159 4 3 1 1 1 1 117 160 8 2 5 1 2 2 118 161 8 1 2 1 2 2 118 162 20 1 4 1 3 3 119 163 3 3 2 1 4 4 120 164 7 2 3 1 3 3 119 165 3 2 3 1 3 3 119 166 3 2 4 1 3 3 119 167 2 1 4 1 4 4 120 168 4 4 3 1 5 5 217 169 20 1 2 1 6 6 218 170 104 1 2 1 4 4 120 171 14 2 2 1 8 8 220 172 7 2 2 1 3 3 119 173 20 4 3 1 3 3 119 174 104 4 3 1 7 7 219 175 14 4 4 1 4 4 120 176 20 4 4 1 2 2 118 177 2 4 5 1 3 3 119 178 111 4 5 1 8 8 220 179 2 4 7 1 4 4 120 180 111 4 7 1 3 3 119 181 2 5 1 1 2 2 118 182 111 5 1 1 7 7 219 183 2 5 2 1 8 8 220 184 111 5 2 1 6 6 218 185 2 4 3 1 4 4 120 186 18 2 4 16 7 7 219 187 5 3 2 2 5 5 217 188 9 4 7 2 13 13 VP 203. 189 9 4 2 2 13 13 VP 203. 190 9 5 3 2 14 14 E.2.238 191 12 1 5 5 8 8 220 192 12 1 7 5 3 3 119 193 9 1 5 5 5 5 217 194 9 1 7 5 11 11 E.fsz.I 195 5 1 5 5 12 12 116 196 5 1 7 5 6 6 218 197 9 4 1 5 16 16 116 198 9 3 2 5 3 3 119 199 300 4 1 5 6 6 218 200 300 4 2 5 8 8 220 201 301 4 4 17 14 14 E.2.238 202 301 4 5 17 15 15 E.3.332 203 200 1 4 4 10 10 S.Asor.S3 204 108 2 5 4 4 4 120 205 4 1 4 18 8 8 220 206 12 2 1 18 10 10 S.Asor.S3 207 12 4 4 18 7 7 219 208 10 1 1 13 13 13 VP 203. 209 10 1 2 13 13 13 VP 203. 210 6 1 7 8 14 14 E.2.238 211 6 1 8 8 5 5 217 212 7 1 9 8 4 4 120 213 7 1 10 8 6 6 218 214 8 1 3 3 7 7 219 215 8 1 4 3 9 9 E.fsz.IV. 216 6 1 3 3 6 6 218 217 6 1 4 3 5 5 217 218 8 2 7 3 2 2 118 219 8 2 8 3 7 7 219 220 1 2 3 10 3 3 119 221 1 3 3 10 13 13 VP 203. 222 1 4 5 10 12 12 116 223 1 5 1 10 10 10 S.Asor.S3 226 2 3 3 19 1 1 117 227 2 2 5 4 12 12 116

2. <span style="font-weight: 600; color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px;">Hány óra van összesen az egyes napokon?</span>

1. Jelenítsük meg a napok nevét és az órák számát az adott napon!
2. Az oszlopokat nevezzük el értelemszerűen!
3. A listát rendezzük óraszám szerint növekvő sorrendbe!

In [8]:
SELECT nap.nap AS 'Nap neve',
       COUNT(ora.ora_id) AS 'Órák száma'
FROM Orak ora JOIN Napok nap ON ora.nap = nap.nap_id
GROUP BY nap.nap
ORDER BY 2

(5 rows affected)

Total execution time: 00:00:00.014

Nap neve Órák száma Péntek 6 Szerda 6 Kedd 13 Csütörtök 19 Hétfő 23

3. <span style="font-weight: 600; color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px;">Készítsünk listát arról, hogy melyik oktatónak hány órája van az Órák táblában!</span>

1. A lista jelenítse meg az oktatók nevét, és az órák számát!
2. Az oszlopokat nevezzük el értelemszerűen!
3. A listából hagyjuk ki az óraadókat!

In [11]:
SELECT *
FROM Orak ora JOIN Oktatok okt ON ora.tanar = okt.oktato_id

(67 rows affected)

Total execution time: 00:00:00.008

ora_id tanar nap Sav targy terem oktato_id nev titulus beosztás status 159 4 3 1 1 1 4 Ujfalussi István Prof, tanszékvezető 6 2 160 8 2 5 1 2 8 Pelikán József NULL 2 3 161 8 1 2 1 2 8 Pelikán József NULL 2 3 162 20 1 4 1 3 20 Kovács András NULL 1 1 163 3 3 2 1 4 3 Szerb László NULL 4 1 164 7 2 3 1 3 7 Kiss János NULL 2 1 165 3 2 3 1 3 3 Szerb László NULL 4 1 166 3 2 4 1 3 3 Szerb László NULL 4 1 167 2 1 4 1 4 2 Szentágotai Istvás Dr. 3 2 168 4 4 3 1 5 4 Ujfalussi István Prof, tanszékvezető 6 2 169 20 1 2 1 6 20 Kovács András NULL 1 1 170 104 1 2 1 4 104 Ludas Mátyás NULL 1 1 171 14 2 2 1 8 14 Szénási ászló NULL 3 1 172 7 2 2 1 3 7 Kiss János NULL 2 1 173 20 4 3 1 3 20 Kovács András NULL 1 1 174 104 4 3 1 7 104 Ludas Mátyás NULL 1 1 175 14 4 4 1 4 14 Szénási ászló NULL 3 1 176 20 4 4 1 2 20 Kovács András NULL 1 1 177 2 4 5 1 3 2 Szentágotai Istvás Dr. 3 2 178 111 4 5 1 8 111 Vitéz János NULL 2 1 179 2 4 7 1 4 2 Szentágotai Istvás Dr. 3 2 180 111 4 7 1 3 111 Vitéz János NULL 2 1 181 2 5 1 1 2 2 Szentágotai Istvás Dr. 3 2 182 111 5 1 1 7 111 Vitéz János NULL 2 1 183 2 5 2 1 8 2 Szentágotai Istvás Dr. 3 2 184 111 5 2 1 6 111 Vitéz János NULL 2 1 185 2 4 3 1 4 2 Szentágotai Istvás Dr. 3 2 186 18 2 4 16 7 18 Nógrádi Kata NULL 1 1 187 5 3 2 2 5 5 Balkán András Dr. 4 2 188 9 4 7 2 13 9 Farkasos Piroska NULL 2 3 189 9 4 2 2 13 9 Farkasos Piroska NULL 2 3 190 9 5 3 2 14 9 Farkasos Piroska NULL 2 3 191 12 1 5 5 8 12 Varga Kata Dr. 4 1 192 12 1 7 5 3 12 Varga Kata Dr. 4 1 193 9 1 5 5 5 9 Farkasos Piroska NULL 2 3 194 9 1 7 5 11 9 Farkasos Piroska NULL 2 3 195 5 1 5 5 12 5 Balkán András Dr. 4 2 196 5 1 7 5 6 5 Balkán András Dr. 4 2 197 9 4 1 5 16 9 Farkasos Piroska NULL 2 3 198 9 3 2 5 3 9 Farkasos Piroska NULL 2 3 199 300 4 1 5 6 300 Szabó Andrera NULL 1 1 200 300 4 2 5 8 300 Szabó Andrera NULL 1 1 201 301 4 4 17 14 301 Juhász András Dr. 6 1 202 301 4 5 17 15 301 Juhász András Dr. 6 1 203 200 1 4 4 10 200 Aranyi Mária NULL 1 1 204 108 2 5 4 4 108 Róbert Gida NULL 2 1 205 4 1 4 18 8 4 Ujfalussi István Prof, tanszékvezető 6 2 206 12 2 1 18 10 12 Varga Kata Dr. 4 1 207 12 4 4 18 7 12 Varga Kata Dr. 4 1 208 10 1 1 13 13 10 Jogrerős Pál Prof. 6 1 209 10 1 2 13 13 10 Jogrerős Pál Prof. 6 1 210 6 1 7 8 14 6 Szép llonka Dr. 3 1 211 6 1 8 8 5 6 Szép llonka Dr. 3 1 212 7 1 9 8 4 7 Kiss János NULL 2 1 213 7 1 10 8 6 7 Kiss János NULL 2 1 214 8 1 3 3 7 8 Pelikán József NULL 2 3 215 8 1 4 3 9 8 Pelikán József NULL 2 3 216 6 1 3 3 6 6 Szép llonka Dr. 3 1 217 6 1 4 3 5 6 Szép llonka Dr. 3 1 218 8 2 7 3 2 8 Pelikán József NULL 2 3 219 8 2 8 3 7 8 Pelikán József NULL 2 3 220 1 2 3 10 3 1 Kovács László Dr. 6 2 221 1 3 3 10 13 1 Kovács László Dr. 6 2 222 1 4 5 10 12 1 Kovács László Dr. 6 2 223 1 5 1 10 10 1 Kovács László Dr. 6 2 226 2 3 3 19 1 2 Szentágotai Istvás Dr. 3 2 227 2 2 5 4 12 2 Szentágotai Istvás Dr. 3 2

4. <span style="color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px;">Van-e olyan tantárgy, amelyből nincs óra?</span>

1. Jelentsük meg a tantárgy nevét és a tantárgyfelelős nevét!

In [35]:
SELECT tan.Tantárgynév,
      -- okt.nev,
       tan.Tkód,
       ora.targy
FROM Orak ora RIGHT JOIN Tantárgyak tan ON ora.targy = tan.Tkód
              --JOIN Oktatok okt ON ora.tanar = okt.oktato_id

(68 rows affected)

Total execution time: 00:00:00.037

Tantárgynév Tkód targy Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Az ürgeöntés alapjai 1 1 Bevezetés az ürgeöntésbe 2 2 Bevezetés az ürgeöntésbe 2 2 Bevezetés az ürgeöntésbe 2 2 Bevezetés az ürgeöntésbe 2 2 Az ürge anatómiája 3 3 Az ürge anatómiája 3 3 Az ürge anatómiája 3 3 Az ürge anatómiája 3 3 Az ürge anatómiája 3 3 Az ürge anatómiája 3 3 Az ürgeöntés hydrológiai alapjai I. 4 4 Az ürgeöntés hydrológiai alapjai I. 4 4 Az ürgeöntés hydrológiai alapjai I. 4 4 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Az ürgeöntés hydrológiai alapjai II. 5 5 Ürgeöntés az EU-ban 8 8 Ürgeöntés az EU-ban 8 8 Ürgeöntés az EU-ban 8 8 Ürgeöntés az EU-ban 8 8 Fejezetek az ürgék jogaiból 10 10 Fejezetek az ürgék jogaiból 10 10 Fejezetek az ürgék jogaiból 10 10 Fejezetek az ürgék jogaiból 10 10 Folyami gáttan 11 NULL Ürgevitel 13 13 Ürgevitel 13 13 ü-business 16 16 Öntési gyakorlat 17 17 Öntési gyakorlat 17 17 Az öntés elmélete 18 18 Az öntés elmélete 18 18 Az öntés elmélete 18 18 ürgefőzés 19 19

5. <span style="background-color: rgb(255, 255, 255); color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px;">Hány óra van összesen a 120-as teremben?</span>

**A 6-10 feladatokat halmazműveletek segítségével oldjuk meg, szintén a tanulmányi adatbázison**

6. <span style="background-color:rgb(255, 255, 255);color:rgb(36, 41, 47);font-family:-apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;;font-size:16px;font-weight:600;">Jelenítsük meg azon tanárok azonosítóját és nevét, akik hétfői vagy keddi napokon tanítanak!</span>

In [33]:
SELECT okt.oktato_id,
       okt.nev
FROM Orak ora JOIN Napok nap ON ora.nap = nap.nap_id
              JOIN Oktatok okt ON ora.tanar = okt.oktato_id
WHERE nap.nap = 'Hétfő'

UNION

SELECT okt.oktato_id,
       okt.nev
FROM Orak ora JOIN Napok nap ON ora.nap = nap.nap_id
              JOIN Oktatok okt ON ora.tanar = okt.oktato_id
WHERE nap.nap = 'Kedd'

(17 rows affected)

Total execution time: 00:00:00.007

oktato_id nev 1 Kovács László 2 Szentágotai Istvás 3 Szerb László 4 Ujfalussi István 5 Balkán András 6 Szép llonka 7 Kiss János 8 Pelikán József 9 Farkasos Piroska 10 Jogrerős Pál 12 Varga Kata 14 Szénási ászló 18 Nógrádi Kata 20 Kovács András 104 Ludas Mátyás 108 Róbert Gida 200 Aranyi Mária

7. <span style="background-color: rgb(255, 255, 255); color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-weight: 600;">Melyek azok a tantárgyak, amelyek esetén van óra a 8.00-kor kezdődő sávban, de nincs a 12.30-kor kezdődő sávban?</span>

8. <span style="background-color: rgb(255, 255, 255); color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-weight: 600;">Listázzuk azokat a termeket, amelyekben oktat Kovács László és van olyan tanóra a teremben, ahol a tantárgy nevében nem szerepel az 'ürge' szó!</span>

9. <span style="font-weight: 600; color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px;">Készítsünk listát arról, hogy melyik tanárnak hány órája van a (munka) hét elején (hétfőn), hét közben (kedd, szerda és csütörtök), illetve a hét végén (péntek)!</span>

1. Jelenítsük meg a tanár azonosítóját, nevét, az időszakot (hét eleje - hét közben - hét vége), valamint az óraszámot!
2. A három megjelölt időszakot külön kérdezzük le, majd fűzzük össze a lekérdezések eredményeit!
3. Rendezzük a listát az oktató neve, azon belül időszak szerint!

10. <span style="background-color: rgb(255, 255, 255); color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-weight: 600;">Melyik napokon nincs órája Pelikán Józsefnek?</span>

In [38]:
--összes nap
SELECT nap
FROM Napok

EXCEPT

--amikor van órája PJ-nek
SELECT nap.nap
FROM Orak ora JOIN Napok nap ON ora.nap = nap.nap_id
              JOIN Oktatok okt ON ora.tanar = okt.oktato_id
WHERE okt.nev = 'Pelikán József'

(3 rows affected)

Total execution time: 00:00:00.016

nap Csütörtök Péntek Szerda

**A 11-15 feladatok gyakorlásra szolgálnak a közelgő zh-ra :-)**

11. _Készítsünk listát az oktatók adatairól!_

1. Az oktatók titulusa és neve összefűzve jelenjen meg, közöttük szóközzel!
2. Ha valakinek nincs titulusa, akkor csak a neve jelenjen meg, szóköz nélkül!
3. A listát rendezzük a beosztás kódja, azon belül a státusz kódja szerint növekvő sorrendbe!
4. Az oszlopokat nevezzük el értelemszerűen!

(335 rows affected)

Total execution time: 00:00:00.009

nap_id nap ora_id tanar nap Sav targy terem 1 Hétfő 159 4 3 1 1 1 1 Hétfő 160 8 2 5 1 2 1 Hétfő 161 8 1 2 1 2 1 Hétfő 162 20 1 4 1 3 1 Hétfő 163 3 3 2 1 4 1 Hétfő 164 7 2 3 1 3 1 Hétfő 165 3 2 3 1 3 1 Hétfő 166 3 2 4 1 3 1 Hétfő 167 2 1 4 1 4 1 Hétfő 168 4 4 3 1 5 1 Hétfő 169 20 1 2 1 6 1 Hétfő 170 104 1 2 1 4 1 Hétfő 171 14 2 2 1 8 1 Hétfő 172 7 2 2 1 3 1 Hétfő 173 20 4 3 1 3 1 Hétfő 174 104 4 3 1 7 1 Hétfő 175 14 4 4 1 4 1 Hétfő 176 20 4 4 1 2 1 Hétfő 177 2 4 5 1 3 1 Hétfő 178 111 4 5 1 8 1 Hétfő 179 2 4 7 1 4 1 Hétfő 180 111 4 7 1 3 1 Hétfő 181 2 5 1 1 2 1 Hétfő 182 111 5 1 1 7 1 Hétfő 183 2 5 2 1 8 1 Hétfő 184 111 5 2 1 6 1 Hétfő 185 2 4 3 1 4 1 Hétfő 186 18 2 4 16 7 1 Hétfő 187 5 3 2 2 5 1 Hétfő 188 9 4 7 2 13 1 Hétfő 189 9 4 2 2 13 1 Hétfő 190 9 5 3 2 14 1 Hétfő 191 12 1 5 5 8 1 Hétfő 192 12 1 7 5 3 1 Hétfő 193 9 1 5 5 5 1 Hétfő 194 9 1 7 5 11 1 Hétfő 195 5 1 5 5 12 1 Hétfő 196 5 1 7 5 6 1 Hétfő 197 9 4 1 5 16 1 Hétfő 198 9 3 2 5 3 1 Hétfő 199 300 4 1 5 6 1 Hétfő 200 300 4 2 5 8 1 Hétfő 201 301 4 4 17 14 1 Hétfő 202 301 4 5 17 15 1 Hétfő 203 200 1 4 4 10 1 Hétfő 204 108 2 5 4 4 1 Hétfő 205 4 1 4 18 8 1 Hétfő 206 12 2 1 18 10 1 Hétfő 207 12 4 4 18 7 1 Hétfő 208 10 1 1 13 13 1 Hétfő 209 10 1 2 13 13 1 Hétfő 210 6 1 7 8 14 1 Hétfő 211 6 1 8 8 5 1 Hétfő 212 7 1 9 8 4 1 Hétfő 213 7 1 10 8 6 1 Hétfő 214 8 1 3 3 7 1 Hétfő 215 8 1 4 3 9 1 Hétfő 216 6 1 3 3 6 1 Hétfő 217 6 1 4 3 5 1 Hétfő 218 8 2 7 3 2 1 Hétfő 219 8 2 8 3 7 1 Hétfő 220 1 2 3 10 3 1 Hétfő 221 1 3 3 10 13 1 Hétfő 222 1 4 5 10 12 1 Hétfő 223 1 5 1 10 10 1 Hétfő 226 2 3 3 19 1 1 Hétfő 227 2 2 5 4 12 2 Kedd 159 4 3 1 1 1 2 Kedd 160 8 2 5 1 2 2 Kedd 161 8 1 2 1 2 2 Kedd 162 20 1 4 1 3 2 Kedd 163 3 3 2 1 4 2 Kedd 164 7 2 3 1 3 2 Kedd 165 3 2 3 1 3 2 Kedd 166 3 2 4 1 3 2 Kedd 167 2 1 4 1 4 2 Kedd 168 4 4 3 1 5 2 Kedd 169 20 1 2 1 6 2 Kedd 170 104 1 2 1 4 2 Kedd 171 14 2 2 1 8 2 Kedd 172 7 2 2 1 3 2 Kedd 173 20 4 3 1 3 2 Kedd 174 104 4 3 1 7 2 Kedd 175 14 4 4 1 4 2 Kedd 176 20 4 4 1 2 2 Kedd 177 2 4 5 1 3 2 Kedd 178 111 4 5 1 8 2 Kedd 179 2 4 7 1 4 2 Kedd 180 111 4 7 1 3 2 Kedd 181 2 5 1 1 2 2 Kedd 182 111 5 1 1 7 2 Kedd 183 2 5 2 1 8 2 Kedd 184 111 5 2 1 6 2 Kedd 185 2 4 3 1 4 2 Kedd 186 18 2 4 16 7 2 Kedd 187 5 3 2 2 5 2 Kedd 188 9 4 7 2 13 2 Kedd 189 9 4 2 2 13 2 Kedd 190 9 5 3 2 14 2 Kedd 191 12 1 5 5 8 2 Kedd 192 12 1 7 5 3 2 Kedd 193 9 1 5 5 5 2 Kedd 194 9 1 7 5 11 2 Kedd 195 5 1 5 5 12 2 Kedd 196 5 1 7 5 6 2 Kedd 197 9 4 1 5 16 2 Kedd 198 9 3 2 5 3 2 Kedd 199 300 4 1 5 6 2 Kedd 200 300 4 2 5 8 2 Kedd 201 301 4 4 17 14 2 Kedd 202 301 4 5 17 15 2 Kedd 203 200 1 4 4 10 2 Kedd 204 108 2 5 4 4 2 Kedd 205 4 1 4 18 8 2 Kedd 206 12 2 1 18 10 2 Kedd 207 12 4 4 18 7 2 Kedd 208 10 1 1 13 13 2 Kedd 209 10 1 2 13 13 2 Kedd 210 6 1 7 8 14 2 Kedd 211 6 1 8 8 5 2 Kedd 212 7 1 9 8 4 2 Kedd 213 7 1 10 8 6 2 Kedd 214 8 1 3 3 7 2 Kedd 215 8 1 4 3 9 2 Kedd 216 6 1 3 3 6 2 Kedd 217 6 1 4 3 5 2 Kedd 218 8 2 7 3 2 2 Kedd 219 8 2 8 3 7 2 Kedd 220 1 2 3 10 3 2 Kedd 221 1 3 3 10 13 2 Kedd 222 1 4 5 10 12 2 Kedd 223 1 5 1 10 10 2 Kedd 226 2 3 3 19 1 2 Kedd 227 2 2 5 4 12 3 Szerda 159 4 3 1 1 1 3 Szerda 160 8 2 5 1 2 3 Szerda 161 8 1 2 1 2 3 Szerda 162 20 1 4 1 3 3 Szerda 163 3 3 2 1 4 3 Szerda 164 7 2 3 1 3 3 Szerda 165 3 2 3 1 3 3 Szerda 166 3 2 4 1 3 3 Szerda 167 2 1 4 1 4 3 Szerda 168 4 4 3 1 5 3 Szerda 169 20 1 2 1 6 3 Szerda 170 104 1 2 1 4 3 Szerda 171 14 2 2 1 8 3 Szerda 172 7 2 2 1 3 3 Szerda 173 20 4 3 1 3 3 Szerda 174 104 4 3 1 7 3 Szerda 175 14 4 4 1 4 3 Szerda 176 20 4 4 1 2 3 Szerda 177 2 4 5 1 3 3 Szerda 178 111 4 5 1 8 3 Szerda 179 2 4 7 1 4 3 Szerda 180 111 4 7 1 3 3 Szerda 181 2 5 1 1 2 3 Szerda 182 111 5 1 1 7 3 Szerda 183 2 5 2 1 8 3 Szerda 184 111 5 2 1 6 3 Szerda 185 2 4 3 1 4 3 Szerda 186 18 2 4 16 7 3 Szerda 187 5 3 2 2 5 3 Szerda 188 9 4 7 2 13 3 Szerda 189 9 4 2 2 13 3 Szerda 190 9 5 3 2 14 3 Szerda 191 12 1 5 5 8 3 Szerda 192 12 1 7 5 3 3 Szerda 193 9 1 5 5 5 3 Szerda 194 9 1 7 5 11 3 Szerda 195 5 1 5 5 12 3 Szerda 196 5 1 7 5 6 3 Szerda 197 9 4 1 5 16 3 Szerda 198 9

<span style="color: #09885a;">12</span>. Készítsünk listát, amely megmutatja, hogy összesen hány tantárgy van kreditenkénti csoportosításban!

  

A listából hagyjuk ki azokat a tételeket, ahol ez a darabszám <span style="color: #09885a;">1</span>!

A Folyami gáttan tantárgyat ne vegyük figyelembe!

A listát rendezzük darabszám szerint csökkenő sorrendbe!

Az oszlopokat nevezzük el értelemszerűen!

13. _Készítsünk listát az oktatók nevéről, a beosztásuk alapján elvárt és a tényleges óraszámukról!_

1. Az oszlopok neve legyen 'Oktató neve', 'Elvárt óraszám' és 'Tényleges óraszám'
2. Szűrjük a listát azokra az oktatókra, ahol az elvárt és a tényleges óraszám azonos

14. _Készítsünk listát, amely megmutatja, hogy mely tantárgyból nincs óra egyáltalán!_

1. Csak a tantárgy neve jelenjen meg!

15. _Hány órájuk van az óraadóknak napi bontásban?_

1. Csak a napok nevét és az óraszámot jelenítsük meg!
2. Rendezzük a listát óraszám szerint növekvő sorrendbe!